wide-graph-with-submodules/run.sh

```diff
@@ -0,0 +1,5 @@
+mkdir -p ./generated
+for i in $(seq 1 1000); do echo module \"module-$i\" \{ source = \"../submodule\" \}; done > ./generated/main.tf
+
+tofu init
```

Comment:
Missing cd ./generated ?

Reply:
`main.tf` pulls this in via `source = "./generated"`.

Reply:
Ah, my mistake!  Was tired last night.